jwst/assign_wcs/util.py

```diff
@@ -1123,9 +1123,9 @@ def compute_footprint_nrs_ifu(dmodel, mod):
     ra_max = np.nanmax(ra_total)
     ra_min = np.nanmin(ra_total)
     # for the footprint we want ra to be between 0 to 360
-    if(ra_min < 0):
+    if (ra_min < 0):
         ra_min = ra_min + 360.0
```

Comment:
```suggestion
    ra_min %= 360.0
```

Reply:
I removed the brackets, but did not change the code since this is just a style PR.